lib/philomena_web/router.ex

```diff
@@ -95,7 +95,9 @@ defmodule PhilomenaWeb.Router do
 
   scope "/api/v1/json", PhilomenaWeb.Api.Json, as: :api_json do
     pipe_through [:accepts_json, :api, :ensure_tor_authorized]
-    resources "/images", ImageController, only: [:show]
+    resources "/images", ImageController, only: [:show] do
+      resources "/comments", CommentController, only: [:show, :index]
```

Comment:
Don't use the same controller for two different resources, even if the result is more/copied code. It makes it much harder to understand.